src/index.html

```diff
@@ -1,13 +1,422 @@
+                <h3 class="testimonial__name">
+                  <a class="link" href="">Max Henry</a>
+                </h3>
+                <div class="testimonial__role">Chief Design Officer</div>
+                <div class="testimonial__about">
+                  When my lovely valley of steam rises and half-day sun is above an impermeable
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section class="page__section contact-us" id="contact-us">
+        <div class="container">
+          <h2 class="page__section-title">
+            Contact us
+          </h2>
+
+          <div class="grid grid--desktop">
+            <div class="grid__item grid__item--1-2">
+              <div class="contact-us__info">
+                <div class="contact-us__info-title">
+                  Phone
+                </div>
+                <a
+                  href="tel:+1 234 5555-55-55"
+                  class="contact-us__info-link"
+                >
+                +1 234 5555-55-55
+                </a>
+              </div>
+
+              <div class="contact-us__info">
+                <div class="contact-us__info-title">
+                  Email
+                </div>
+                <a
+                  href="mailto:hello@miami.com"
+                  class="contact-us__info-link"
+                >
+                hello@miami.com
+                </a>
+              </div>
+            </div>
+
+            <div class="grid__item grid__item--4-5">
+              <div class="contact-us__info contact-us__info--last">
+                <div class="contact-us__info-title">
+                  Adress
+                </div>
+                <a
+                  href="https://goo.gl/maps/MTtJeNQcynKRPBmr7"
+                  class="contact-us__info-link"
+                  rel="noopener"
+                  target="_blank"
+                >
+                  400 first ave. n.
+                  <br>
+                  suite 700
+                  <br>
+                  Minneapolis, MN 55401
+                </a>
+              </div>
+            </div>
+
+            <div class="grid__item grid__item--8-12">
+              <form action="">
+                <input
+                  name="name"
+                  type="text"
+                  placeholder="Name"
+                  class="contact-us__field form-field"
+                  required
+                >
+                <input
+                  name="email"
+                  type="email"
+                  placeholder="Email"
+                  class="contact-us__field form-field"
+                  required
+                >
+                <textarea
+                  name="message"
+                  placeholder="Message"
+                  class="contact-us__field contact-us__field--last form-field form-field--textarea"
+                  required
+                >
+                </textarea>
```

Comment:
```suggestion
               ></textarea>
```
З textarea зараз існує баг. Немає плейсхолдера і текст вводиться в неправильному місці. Щоб це пофіксити, заміни форматування саме цього тега, як ми з Миколою вказали (закриваюча і відкриваюча дужка на одному рядку, і ОДИН відступ вліво)


https://user-images.githubusercontent.com/66570142/136373431-b2826b08-8796-4505-9e15-10370b97da8c.mov